README.md

```diff
@@ -3,18 +3,26 @@
 ## Getting Started
 
 1. [Install Docker](https://docs.docker.com/get-docker/)
-2. Clone this repo `git clone git@github.com:Cant-Stop-Columbus/vaccinateoh.git`
-3. Create a `build/env.mk` file with at least a value for your local domain (ensure the hostname you use is pointed to your machine) and avoid pulling images from a docker repo:
+1. [Install docker-compose](https://docs.docker.com/compose/install/)
+1. [Install composer](https://getcomposer.org/download/)
+1. Clone this repo `git clone git@github.com:Cant-Stop-Columbus/vaccinateoh.git`
+1. Create a `build/env.mk` file with at least a value for your local domain (ensure the hostname you use is pointed to your machine) and avoid pulling images from a docker repo:
 ```
 export LOCAL_DOMAIN=vaccinateoh.local
 ```
-4. Start nginx-proxy `docker run -d -p 80:80 -v /var/run/docker.sock:/tmp/docker.sock:ro jwilder/nginx-proxy`
-5. `cd build` before the remaning steps.
-6. Start your local web, db, mailhog, and pgadmin containers with `make up`
-7. Run migrations with `make migrate seed-locations`
-8. Run your app assets build process by entering a shell on the web container with `make watch`
-9. Visit the site in your browser at https://vaccinateoh.local (or the domain you selected above).
+6. `cd src/`
+1. `composer update` - the syntax for this step will depend on how you installed `composer`
+1. `composer install` - the syntax for this step will depend on how you installed `composer`
+1. `cd ../build`
```

Comment:
We can avoid this by running the `composer install` inside the web container. I will add a `make install` command to the Makefile to help with this. Could you remove these lines and change it to just `cd build`? I'll submit a PR with my changes after we get this one approved and merged.

Reply:
Done!